### PR TITLE
Add Git and GitHub content.

### PR DESCRIPTION
**Pull request by chrisrhoton**

### CODE REVIEW

File: tracks/dev-tools/git-and-github/index.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Git & GitHub
+
+* ["Getting Started - About Version Control"](http://git-scm.com/book/en/v2/Getting-Started-About-Version-Control) via [Git's website](http://git-scm.com/). #article
+* ["Git Basics: What is Version Control"](http://git-scm.com/video/what-is-version-control) via [Git's website](http://git-scm.com/). #video
+* ["Git Basics: What is Git?"](http://git-scm.com/video/what-is-git) via [Git's website](http://git-scm.com/). #video
+* [Try Git](https://try.github.io/) - "Got 15 minutes and want to learn Git?" #tutorial
+* [Git Immersion](http://gitimmersion.com/) by [Neo Innovation Inc](http://neo.com/). #tutorial
+* [Everyday Git](http://git-scm.com/docs/everyday) via [Git's website](http://git-scm.com/). #article
+* [Ry's Git Tutorial](http://rypress.com/tutorials/git/index) by [RyPress](http://rypress.com/). #tutorial
+* ["GitHub for Beginners: Don't Get Scared, Get Started"](http://readwrite.com/2013/09/30/understanding-github-a-journey-for-beginners-part-1)) by [Lauren Orsini](https://twitter.com/laureninspace). #article
```
Comment (ericelliott) [inline tracks/dev-tools/git-and-github/index.md]: It looks like a lot of these might cover the same material. How many Git articles does it take to screw in a lightbulb.

Imagine you're talking a class on Git. Would you want to be exposed to all of this material in exactly this order?

Pick your favorites. No more than 2-3 from each category. They should all work well in sequence from first to last. (One picks up where the other left off, maybe diving into a bit more depth, or explaining a common workflow, etc...).

File: README.md [hunk 2]
```diff
@@ -26,8 +26,8 @@ We're trying to collect the best free training material in the world, so we have
 ## Contents
 1. [Fundamentals](tracks/fundamentals/index.md)
 1. [Developer Tools](tracks/dev-tools/index.md)
-  1. [Git & GitHub](tracks/git-and-github/index.md)
-  1. [Command Terminal](tracks/terminal/index.md)
+  1. [Git & GitHub](tracks/dev-tools/git-and-github/index.md)
```
Comment (ericelliott) [inline README.md]: Good catch!

**Comment by chrisrhoton** [comment]
Updated with better learning flow + stricter curation.

**Comment by ericelliott** [comment]
Much better. :100:

**Pull request merged**